emacs/merlin-imenu.el

```diff
@@ -15,14 +15,10 @@
 (require 'subr-x)
 (require 'merlin)
 
-;;; enable depth and size threshold for OCaml modules with big size
-(setq max-lisp-eval-depth 10000)
-(setq max-specpdl-size 10000)
-
 ;; lists of different outline items
-(defvar-local value-list nil)
-(defvar-local type-list nil)
-(defvar-local exception-list nil)
+(defvar-local merlin-imenu-value-list nil)
+(defvar-local merlin-imenu-type-list nil)
+(defvar-local merlin-imenu-exception-list nil)
```

Comment:
If we're going to rename them, shall we just make them private using the `--` convention? (I don't know, I trust you as the elisp expert here)

Reply:
Agreed, we should do that. I'll update the PR.